### PR TITLE
bgrzesik-dev to main

### PR DESCRIPTION
**Pull request by bgrzesik**

### CODE REVIEW

**Review by bzapa — APPROVED** [review]

**Comment by bgrzesik** [comment]
@bzapa Done. If changes do not require reviewing again, please rebase and merge.

File: server/files/handler.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    def __init__(self):
+        Actor.__init__(self)
+        self._store = {}
+        self._next_upload_id = 0
+        self._uploads = {}
+
+    @Actor.assert_executor()
+    def start_upload(self, name):
+        logging.debug(f"start_upload(): name={name}")
+        new_file = UploadedFile(name, rw=True)
```
Comment (bzapa) [inline server/files/handler.py:51]: path prefix should be added

**Review by bzapa — APPROVED** [review]
Doesn't work. Is it supposed to? No useful logs

File: server/files/handler.py (hunk 1 above)
Comment (bgrzesik) [reply server/files/handler.py:51]: I agree and additional checks, but I wanted to push out those changes as soon as possible.

**Comment by bgrzesik** [comment]
> Doesn't work. Is it supposed to?

Could you elaborate?

**Comment by bzapa** [comment]
> > Doesn't work. Is it supposed to?
> 
> Could you elaborate?

I tested it with your current branch on mobile. It didn't work and I couldn't find useful logs. So idk if it's not finished yet or chosen file didn't meet some constraints.

**Comment by bgrzesik** [comment]
> > > Doesn't work. Is it supposed to?
> > 
> > 
> > Could you elaborate?
> 
> I tested it with your current branch on mobile. It didn't work and I couldn't find useful logs. So idk if it's not finished yet or chosen file didn't meet some constraints.

I updated both branches, but I see mobile repo doesn’t compile after rebase, could you try fixing it and test it?

File: server/files/handler.py (hunk 1 above)
Comment (bgrzesik) [reply server/files/handler.py:51]: Let’s merge and add this with configuration

**Pull request merged**